model/survey-instance.js

```diff
@@ -7,17 +7,21 @@
 const Sequelize = require('sequelize');
 
 /**
- * Each Survey-instance consists of many survey instances
- * @typedef {Object} SurveyInstance
- * @property {String} name - SurveyInstance name
- */
-
-/**
  * Registers model with Sequelize
  * @param {Sequelize} sequelize - database instance
  * @returns {Null} nothing
  */
 function register (sequelize) {
+    /**
+     * Each SurveyInstance consists of many QuestionResults
+     * @typedef {Object} SurveyInstance
+     * @property {String} name - SurveyInstance name
+     * @property {Date} startTime - when the survey is availible
+     * @property {Date} endTime - when the survey must be completed
+     * @property {Date} userSubmissionTime - time that the server recieved the submitted survey
+     * @property {Date} actualSubmissionTime - time that patient submitted survey on application
```

Comment:
User submission time - time user submitted the survey in the app.
Actual submission time - the time survey instance was marked submitted in the database.